src/notifications/generate.ts

```diff
@@ -34,7 +34,7 @@ const systemTitle = () => undefined;
 
 export const notificationTitleMap: Record<
   NotificationType,
-  (ctx: NotificationBaseContext) => string | undefined
+  (ctx: never) => string | undefined
```

Comment:
I've cast this as never because that is the only way (that I know of) to get dynamic types on `ctx` params in the `notificationTitleMap` below. Only caveat to this is that we have to cast to `never` in `generateNotificationV2` but think its better then casting in this file since `generateNotificationV2` is more internal function